pkg/translator/zipkin/zipkinv2/from_translator.go

```diff
@@ -178,6 +170,30 @@ func spanToZipkinSpan(
 	return zs, nil
 }
 
+func populateStatus(status pdata.SpanStatus, zs *zipkinmodel.SpanModel, tags map[string]string) {
+	if status.Code() == pdata.StatusCodeError {
+		tags[tracetranslator.TagError] = "true"
+	} else {
+		// The error tag should only be set if Status is Error. If a boolean version
+		// ({"error":false} or {"error":"false"}) is present, it SHOULD be removed.
+		// Zipkin will treat any span with error sent as failed.
+		delete(tags, tracetranslator.TagError)
+	}
+
+	// Per specs, Span Status MUST be reported as a key-value pair in tags to Zipkin, unless it is UNSET.
+	// In the latter case it MUST NOT be reported.
+	// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/zipkin.md#status
```

Comment:
@codeboten this is how I read the specs here - if Status is unset, no status field (neither code nor description) should be reported. What do you think?

Reply:
That matches my expectation, thanks for fixing!